README.md

```diff
@@ -38,6 +38,7 @@ NOTE: Database is SQLite3 via SQLAlchemy
     - View detailed coverage reports, with listings for untested lines of code ...
       - As a web page: `pipenv run python view_coverage.py`
       - In the console: `pipenv run view_coverage`
+    - Tests can be run automatically after each file save using [pytest-watch](https://pypi.org/project/pytest-watch/). Visit the documentation to learn how to run it for your system. See [PR #72](https://github.com/codeforpdx/dwellinglybackend/pull/72) for a preview of what it can do.
```

Comment:
Can we put some examples in on how to run the automatic runner for Windows and Mac?

Reply:
I'll just merge this now, since it seems like there are a few different ways of running the test watcher